common/ShittyUtils.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#define MIN(a,b) ((a)<(b)?(a):(b))
```

Comment:
std::min exists, so not sure if this is necessary
https://en.cppreference.com/w/cpp/algorithm/min

Reply:
good catch, fixing it